doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md

```diff
@@ -26,10 +26,10 @@ Colum should be provisioned as type [`geometry`](http://postgis.net/workshops/po
 [Top](#top)
 
 ## <a name="section2"></a>Administration guide
-Please refear to [`NGSIPostgreSQLSink`](/ngsi_postgresql_sink.md) since all administration options about PostgreSQLSink applies to PostGISSink.
+Please refear to [`NGSIPostgreSQLSink`](/doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md) since all administration options about PostgreSQLSink applies to PostGISSink.
```

Comment:
Maybe better to use a relative path? Ie.:

```suggestion
Please refear to [`NGSIPostgreSQLSink`](ngsi_postgresql_sink.md) since all administration options about PostgreSQLSink applies to PostGISSink.
```